setup/ci/gcb.md

```diff
@@ -73,3 +77,46 @@ if the build produces a Docker image, you can add an expected artifact of type *
 `gcr.io/my-project-id/my-application` (replacing `my-project-id` and `my-application` with
 appropriate values). You can then [use the produced image](/reference/artifacts/in-pipelines/)
 in downstream stages.
+
+## Configure a Google Cloud Build Stage
+
+To run a Google Cloud Build as part of a Spinnaker pipeline, create a stage of type *Google Cloud Build*.  Configure the
+stage by selecting the GCB account to use to run the build, and entering the [build configuration YAML](https://cloud.google.com/cloud-build/docs/build-config)
+in the provided text box:
+![](/setup/ci/gcb_config.png)
+You may also provide the build definition YAML as an artifact.
+
+In the *Produces Artifacts* section, you may supply any artifacts that you expect the build to create in order to
+make these artifacts available to downstream stages.  Google Cloud Build supports creating either GCS or Docker image
+[artifacts](https://cloud.google.com/cloud-build/docs/configuring-builds/store-images-artifacts), either of which
+will be converted to Spinnaker artifacts and injected into the pipeline on completion of the build.
+
+While your build is executing, the stage details will provide the current status of the build and a link to view
+the build logs in the Google Cloud Console:
+![](/setup/ci/gcb_status.png)
+
+## Configuration via Pub/Sub
+
+Prior to version 1.14, Spinnaker did not have built-in support for Google Cloud Build, but pipelines could be
+triggered on changes to the build status by directly listening on the Pub/Sub subscription.
+
+    ```
+    hal config pubsub google subscription add $PUBSUB_SUBSCRIPTION_NAME \
+      --project $PROJECT_ID \
+      --subscription-name $SUBSCRIPTION_NAME \
+      --message-format GCB
+
+    hal config pubsub google enable
+
+    hal deploy apply
+    ```
+
+The steps to create a pipeline trigger in this case are exactly the same except that the **Subscription Name** field
```

Comment:
exactly the same as configuring the GCB stage? or same as what?

Reply:
I'll change this to a link directly to the above instructions so it's clear